src/Stripe.net/Entities/_contracts/ISupportMetadata.cs

```diff
@@ -1,12 +0,0 @@
-namespace Stripe
```

Comment:
Wasn't this used to ensure we added `Metadata` to the Resource too?

Reply:
I'm not sure why resource classes used this interface at all, but I don't think this is a good reason: the interface would only force you add the `Metadata` property if you remembered to add the interface...